RecoParticleFlow/PFTracking/plugins/GoodSeedProducer.cc

```diff
@@ -222,11 +220,11 @@ GoodSeedProducer::produce(Event& iEvent, const EventSetup& iSetup)
 	  int ipteta=getBin(Tk[i].eta(),Tk[i].pt());
 	  int ibin=ipteta*9;
 	  
-	  float oPTOB=1.f/Tj[i].lastMeasurement().updatedState().globalMomentum().mag();
```

Comment:
in my opinion this is a defect of the original code: PTOB seems to ask for the outermost state not the innermost